### PR TITLE
Add maintenance window for Covid-19 Vaccination app

### PR DESCRIPTION
**Pull request by stephenBarrs**
## Description
Adds maintenance windows on Call to Action, Introduction and Form pages as both can be accessed directly with the URL. 

## Testing done
Manual verification in Review Instance

## Screenshots
Call to Action page:
image

intro page:
image

form page:
image

## Acceptance criteria
- [ ] Form intro page shows maintenance banner when maintenance window is active in Pager Duty
- [ ] Form page shows maintenance banner when maintenance window is active in Pager Duty
-
## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by billfienberg — APPROVED** [review]

File: src/applications/coronavirus-vaccination/components/Introduction.jsx [hunk 1]
```diff
@@ -22,75 +27,80 @@ function Introduction({ isLoggedIn, toggleLoginModal }) {
...
-              }}
-            >
-              Continue without signing in
-            </Link>
-          </p>
-        </>
-      )}
+      <DowntimeNotification
+        appTitle="Covid 19 Vaccination Information"
+        dependencies={[externalServices.vetext_vaccine]}
```
Comment (ncksllvn) [inline src/applications/coronavirus-vaccination/components/Introduction.jsx:32]: I think the dependency here should be in camel case -> `externalServices .vetextVaccine`

Comment (stephenBarrs) [reply src/applications/coronavirus-vaccination/components/Introduction.jsx:32]: Yep.  the devops config is with _ and I originally cut and pasted that.  thanks!

**Comment by batemapf** [comment]
@stephenBarrs is it possible (if not done already and just not screen shotted) to alter the text in the blue box on /health-care/covid-19-vaccine in a maintenance window situation? E.g.

Screen Shot 2021-01-07 at 3 30 27 PM

**Comment by CEDAR-Evaluator** [comment]
I believe we have a standard outage message UI pattern....? Let me find out.

**Comment by ncksllvn** [comment]
I think the usual pattern is that we show a yellow alert box with this content -

```
title: This tool is down for maintenance.
body: We’re making some updates to this tool to help make it even better for
        Veterans, service members, and family members like you. We’re sorry it’s
        not working right now. Please check back soon.
```

**Comment by rtwell** [comment]
i think @rianfowler has in depth knowledge about how we handle these, if it is helpful

**Comment by stephenBarrs** [comment]
@batemapf Using the normal UI pattern with the yellow bar (seen in screenshots above) would result in the outage box inside the Stay Informed box, which would be pretty weird.  

@rianfowler are there examples of apps not using the standard UI Pattern and just hiding a button and displaying outage text without displaying the yellow alert box?

**Comment by ncksllvn** [comment]
> @stephenBarrs is it possible (if not done already and just not screen shotted) to alter the text in the blue box on /health-care/covid-19-vaccine in a maintenance window situation? E.g.

Just need to use the `render` prop -> 

```jsx
function renderDowntime(downtime, children) {
  if (downtime.status === 'down') {
    return <button disabled>Can't do that right now</button>;
  }
  return children; // Render normal enabled button
}

<DowntimeNotification
        dependencies={[vettextVaccine]}
        render={renderDowntime}
      >
      <button>Go to the form</button>
</DowntimeNotification>
```

**Review by batemapf — APPROVED** [review]

**Pull request merged**